### ISSUE TITLE
Video and audio not in sync

### ISSUE THREAD

**Issue by amilajack** [duanyao/codecbox.js#4]
When playing files that have a long duration, the audio and video become out of sync.

**Comment by duanyao** [comment]
Yes, it's a known problem. At the time I wrote this code, I have little experience in writing a video player, so the player just renders frames at the pace of FPS specified by the video file and doesn't try to sync with audio at all. I'm reading this [tutorial](http://dranger.com/ffmpeg/tutorial05.html) recently and will see if I can fix it.
